### DISCUSSION TITLE
Raspberry Pi binary

### DISCUSSION THREAD

**Discussion by TDStrasser** [Serial-Studio/Serial-Studio#37] [Ideas]
The Raspberry Pi is a very popular and affordable computing platform, running its own distinct version of Linux.  This is of particular value in STEM instruction settings.  Having a downloadable binary of Serial Studio available for distribution for the Raspberry Pi with the Raspbian operating system would be very convenient and not require Raspberry Pi users among them teachers and STEM students who are not versed in the intricacies of the Qt framework and compilers to build the application every time there is a release, even if they knew how.

So it would be wonderful if you could offer a Raspberry Pi binary as part of your distribution.  I would be very popular among STEM students, their teachers and makers in general..

**Comment by alex-spataru** [comment]
Hi, please check [this](https://github.com/Serial-Studio/Serial-Studio/discussions/81#discussioncomment-1896487) discussion for compiling and installing Serial Studio on the Raspberry Pi. I was only able to test on a virtual machine running Raspberry Pi OS 2021-01-11, so I cannot guarantee that Serial Studio works on real hardware.

Happy new year!

**Comment by TDStrasser** [comment]
Thank you Alex and happy new year.
I’ll try this during next week as I’ll be away this week.
I also use Qt5 but in Python as Pyside2 and have been able to deploy apps for Windows, Mac and Raspbian. I use the real hardware though.
I’ll let you know how it goes. Thanks for your time and effort on this.
Kind regards
Tony
